wscript

```diff
@@ -1,15 +1,13 @@
 # -*- python -*-
 
 
-from subprocess import getstatusoutput
+import subprocess
 
 from waflib import Utils
 
-
-# slight code duplication with hamster/__init__.py, but this is finally cleaner.
-rc, output = getstatusoutput("git describe --tags --always --dirty=+")
-VERSION = "3.0.2" if rc else output
-
+# Reuse code from hamster to figure out the version number to use
+process = subprocess.run(["python3", "src/hamster/version.py"], check=True, stdout=subprocess.PIPE, text=True)
```

Comment:
This commit (specifically the `text=True`) broke build on openSUSE Leap 15.x, which still ships python 3.6 as standard python version.

Yes I know it's outdated, but this is how LTS linux works.

Reply:
Ah, we also claim to support 3.6, so that should be fixed. There is probably an older API that could work (maybe passing `encoding='UTF-8'`?), or otherwise omitting that and just calling `.decode()` on the binary string that will then be returned.

I've added an item to #747, since that already has an item regarding `VERSION` parsing.

Reply:
(unfortunately github workflows does not have an Ubuntu with python3.6 available anymore, so we cannot easily verify py3.6 compatibility...)

Reply:
OK, I see you added this to a new ticket (#747), likely to become the reference for "fixed for 3.0.4". So we can keep this one closed.

Reply:
I simply patched out the "text=True" for the Leap build. It worked.

Reply:
Are you sure? I think I got `VERSION=b'3.0.3'` in the resulting defs.py without the text argument, but I'm not entirely sure.